topics/data-science/tutorials/python-math/tutorial.md

```diff
@@ -0,0 +1,359 @@
+---
+layout: tutorial_hands_on
```

Comment:
pretty good first tutorial for new programmers.
I only encountered a small bug which i have a fix in the comments
also great work with showing there are multiple ways to Rome

maybe for later if you are really bored.
a build in check if their answer is correct (* but they can also just check the solution )